ros2cli_test_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2cli_test_msgs</name>
+  <version>1.0.1</version>
+  <description>A package containing interface definitions for testing ros2cli.</description>
+  <maintainer email="audrow@openrobotics.org">Audrow Nash</maintainer>
```

Comment:
I guess we should also add maintainer tags for other maintainers of the ros2cli repository.